### PR TITLE
Update color textures for Uma

### PR DESCRIPTION
**Pull request by gergahn**
Screenshot 2024-02-27 at 15 14 34
Screenshot 2024-02-27 at 15 14 39
Screenshot 2024-02-27 at 15 15 16
Screenshot 2024-02-27 at 15 15 23

### CODE REVIEW

File: client/ProjectSettings/EditorBuildSettings.asset
Comment (AngieDutra) [inline client/ProjectSettings/EditorBuildSettings.asset:1]: I have a question on why was this modified

Comment (AngieDutra) [reply client/ProjectSettings/EditorBuildSettings.asset:1]: I'll restore this ✨

**Review by AngieDutra — APPROVED** [review]
✔️ Tested on Android Samsung s23+
✔️ Checked the heriarchy and the files are well placed
✔️ Both in UI and in game the new colors look nice

**Review by BertovDev — APPROVED** [review]
